intermediate-javascript-calorie-counter/index140.js

```diff
@@ -0,0 +1,14 @@
+document.getElementById('calorie-form').onsubmit = calculate;
+
+function calculate(e) {
+  e.preventDefault();
+
+  const total = Array.from(document.getElementsByClassName('cal-control')).map(
+    meal => meal.value
```

Comment:
The `Number()` function bit of this lesson could be incorporated into the way I suggested to split the lessons above.